File: Salon-Owner-Service/src/main/java/com/stackroute/SalonOwnerService/model/SalonOwner.java
package com.stackroute.SalonOwnerService.model;

import lombok.Data;
import org.springframework.data.annotation.Id;
import org.springframework.data.elasticsearch.annotations.Document;

import java.util.List;
@Data
@Document(indexName= "SalonOwner")
public class SalonOwner {
    // Salon Owner Properties
    @Id
    String ownerId;
    String email;
    String name;
    String mobileNo;
    String referalCode;
    String addr;
    String dob;
    List<Salon>salonList;
    boolean aprovedbyadmin ;

    public boolean isAprovedbyadmin() {
        return aprovedbyadmin;
    }

    public void setAprovedbyadmin(boolean aprovedbyadmin) {
        this.aprovedbyadmin = aprovedbyadmin;
    }

    public List<Salon> getSalonList() {
        return salonList;
    }

    public void setSalonList(List<Salon> salonList) {
        this.salonList = salonList;
    }
}

File: product-webapp/webapp/src/app/model/SalonOwner.ts
import { Salon } from './Salon';

export class SalonOwner {
  ownerId?: String;
  email?: String;
  name?: String;
  mobileNo?: String;
  referalCode?: String;
  addr?: String;
  dob?: String;
  salonList?: Array<Salon>;
  aprovedbyadmin?: boolean;
  price?: String;

}
